awx/api/views.py

```diff
@@ -3142,7 +3142,10 @@ def post(self, request, *args, **kwargs):
             data = OrderedDict()
             data['job'] = new_job.id
             data['ignored_fields'] = self.sanitize_for_response(ignored_fields)
-            data.update(JobSerializer(new_job, context=self.get_serializer_context()).to_representation(new_job))
+            if isinstance(new_job, WorkflowJob):
+                data.update(WorkflowJobSerializer(new_job, context=self.get_serializer_context()).to_representation(new_job))
```

Comment:
In this case, I can see the `job` field confusing people. I might prefer to make that `None`, or else, if the jobs have already been spawned return a field `jobs` that lists them. I don't think we need to include a literal message explaining what happened, I think the current data structure could act as a red herring for people writing clients.